### PR TITLE
[new] Adding Support Center docs

### PR DESCRIPTION
**Pull request by angelaLazar**
### Your checklist for this pull request

Please review the [guidelines for contributing](https://github.com/auth0/docs#contributing) to this repository.
- [x] Commit messages conforms to our [commit message standards](https://github.com/auth0/docs#commit-messages)
- [x] Content conforms to our [Contributing Guidelines](https://github.com/auth0/docs#contributing-guidelines)
- [x] If applicable, you have added details to the [update feed](https://github.com/auth0/docs/tree/master/updates) 
### Description

Adding docs for Support Center, a page on how to create an Issue and how to choose the correct issue type.

### CODE REVIEW

**Comment by ntotten** [comment]
@solepano and @yvonnewilson can you give this a quick review

File: articles/support/issue-types.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Choosing the Issue Type
```
Comment (ntotten) [inline articles/support/issue-types.md]: I think this might make sense to just be on the same page as opening a ticket

**Comment by solepano** [comment]
It looks good to me! I will add a comment saying that tickets opened tru support center will be seen to all account admins, and that any admin can add a comment on an existing ticket.

Regarding the "mark as solved" feature. Maybe clarify that is only available under some cinditions (if the user is the ticket requester, the ticket is assigned to an agent and it is not solved or closed already)

**Comment by angelaLazar** [comment]
Addressed comments in latest commit

**Pull request merged**